microsoft-edge/webview2/release-notes/index.md

```diff
@@ -16,18 +16,107 @@ These Release Notes provide information about new features and bug fixes that ar
 Sep 2024
-Aug 2024
 
 moved to archive:
-Jul[/Aug] 2024
+Aug 2024
 -->
 
 
+<!-- Feb 2025 Release SDK -->
+<!-- ====================================================================== -->
+## 1.0.3065.39
+
+Release Date: February 10, 2025
+
+[NuGet package for WebView2 SDK 1.0.3065.39](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3065.39)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 133.0.3065.39 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+No additional APIs have been promoted to Stable and added in this Release SDK.
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Added the missing **Close** button in the **Download** flyout.
+* Fixed a race condition that occurred when the Web Request Response event never occurs.
+
+<!-- end of Feb 2025 Release SDK -->
+
+
+<!-- Feb 2025 Prerelease SDK -->
+<!-- ====================================================================== -->
+## 1.0.3116-prerelease
+
+Release Date: February 10, 2025
+
+[NuGet package for WebView2 SDK 1.0.3116-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3116-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 134.0.3116.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+No Experimental APIs have been added in this Prerelease SDK.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+No APIs have been promoted from Experimental to Stable in this Prerelease SDK.
+
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Added the missing **Close** button in the **Download** flyout.
+* Fixed a race condition that occurred when the Web Request Response event never occurs.
+
+
+<!-- ---------- -->
+###### SDK-only
+
+* Fixed .NET and Win32 documentation of the `CoreWebView2Find.FindNext` method that incorrectly mentioned `FindPrevious`.  The method summary now mentions `FindNext` instead.  ([Issue #5059](https://github.com/MicrosoftEdge/WebView2Feedback/issues/5059))
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2Find.FindNext Method](/dotnet/api/microsoft.web.webview2.core.corewebview2find.findnext?view=webview2-dotnet-1.0.3116-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2Find.FindNext Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2find?view=webview2-winrt-1.0.3116-prerelease&preserve-view=true#findnext)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalFind::FindNext](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalfind?view=webview2-1.0.3116-prerelease&preserve-view=true#findnext)
+<!-- todo: make sure Win32 API Ref 3116 has "Next" not "Previous" in summary -->
```

Comment:
has Win32 API Ref been fixed to say "Next" instead of "Previous" in summary?
@sivMSFT 

RelNotes linking to live Ref docs:
https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#customize-the-find-behavior-find-api
.NET - "Previous"; incorrect.
WinRT - "Next"; correct.
.NET - "Previous"; incorrect.